.circleci/config.yml

```diff
@@ -67,6 +67,48 @@ jobs:
           name: Check generate
           command: make check-generate
 
+  test-stable-kubernetes:
+    machine: true
+    resource_class: xlarge
+    environment:
+      K8S_VERSION: v1.10.0
+      KUBECONFIG: /home/circleci/.kube/config
+      MINIKUBE_VERSION: v0.30.0
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      MINIKUBE_HOME: /home/circleci
+      CHANGE_MINIKUBE_NONE_USER: true
+    steps:
+      - checkout
+      - run:
+          name: setup kubectl
+          command: |
+            curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
Can you enlighten me how a developer would run these tests locally? Are there docs for this? Even better, is there a way to embed this into the current workflow?

Reply:
All of the kubernetes tests are behind a `kubernetes-integration-tests` feature flag and not enabled by default.

```
minikube start
cargo test --features kubernetes-integration-tests
```

It will use your kube config that exists in `~/.kube/config`. Minikube will place one for you or you can run this command to fetch the config from an existing cluster. It just needs _some_ kube cluster.
```
curl http://<some kube cluster>/config > ~/.kube/config
```

The idea here is that if you have access to a kube cluster via `kubectl` then you can run the tests.

Reply:
Where is this documented?

Reply:
Happy to add this to the contributing guide, will do that tomorrow. We can also provide a separate test command to test with minikube.